service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -612,8 +612,8 @@ public void setTariffLimitByAmountOfBags(Long tariffId, EditAmountOfBagDto dto)
     public void setTariffLimitBySumOfOrder(Long tariffId, EditPriceOfOrder dto) {
         TariffsInfo tariffsInfo = tryToFindTariffById(tariffId);
         tariffsInfo.setCourierLimit(CourierLimit.LIMIT_BY_SUM_OF_ORDER);
-        tariffsInfo.setMaxPriceOfOrder(dto.getMaxPriceOfOrder());
-        tariffsInfo.setMinPriceOfOrder(dto.getMinPriceOfOrder());
+        tariffsInfo.setMinQuantity(dto.getMaxPriceOfOrder());
+        tariffsInfo.setMaxQuantity(dto.getMinPriceOfOrder());
```

Comment:
same here

Reply:
Is this logic covered by the test?
If it is the test should fail

Reply:
I see you removed `minPriceOfOrder` in several `DTOs`
Should it be removed in `EditPriceOfOrder` too?
Same about `maxPriceOfOrder`

Reply:
done